client/src/js/views/IndexTemplate.html

```diff
@@ -1,54 +1,81 @@
+            <div class="hero__content-row">
+                <div class="hero__copy columns small-12 large-6">
+                    <h1 class="hero__title">
+                        <a href="/#">
+                            <object class="logo" type="image/svg+xml" data="/img/paw_wordmark.svg"></object>
+                            <span class="show-for-sr">Paw</span>
+                        </a>
+                    </h1>
+                    <p class="hero__text">
+                        PAW &ndash; <em>puppies at work</em> &ndash; a slack-integrated tracking
+                        system that keeps tabs of dogs in the workplace.
+                        PAW... unleash your dog.
+                    </p>
+                    <a class="jump-to-watch hero__button button button--shikoba button--hover-fill-primary" href="#watch">
+                  
+                        <object class="button__icon" type="image/svg+xml" data="/img/isl-paw-play-button--white.svg"></object>
+                        <span>Watch Video</span>
                     </a>
-                </h1>
-                <p class="hero__text">
-                    PAW &ndash; <em>puppies at work</em> &ndash; a slack-integrated tracking
-                    system that keeps tabs of dogs in the workplace.
-                    PAW... unleash your dog.
-                </p>
-                <a class="jump-to-watch hero__button button" href="#watch">Watch Video</a>
-            </div>
-        </div>
-        <div class="decoration">
-            <div class="row">
-                <div class="columns">
+                </div>
+                <div class="columns decoration__workspace-container">
                     <object class="decoration__workspace" type="image/svg+xml" data="/img/workspace.svg"></object>
-                    <a class="jump-to-watch decoration__scroll icon icon--down show-for-medium" href="#watch">
-                        <span class="show-for-sr">Scroll Down</a>
-                    </a>
                 </div>
             </div>
         </div>
+        <div class="decoration__floor">
+            
+        </div>
+
+        <div class="decoration__scroll">
+            <a class="jump-to-watch  icon icon--down show-for-medium" href="#watch">
+                <img type="image/svg+xml" src="/img/down-arrow.svg">
+                <span class="show-for-sr">Scroll Down</a>
+            </a>
+        </div>
+       
     </section>
 
     <section class="video" id="watch">
-        <a class="video__play button button--alt" href="https://vimeo.com/134892537" data-media="https://player.vimeo.com/external/134892537.hd.mp4?s=10bfa368f8bffb4a4a09f6abec6c96e770aa2e02&profile_id=113" target="_blank">Watch Video</a>
+       
+
+        <a class="video__play button button--shikoba button--alt" href="https://vimeo.com/134892537" data-media="https://player.vimeo.com/external/134892537.hd.mp4?s=10bfa368f8bffb4a4a09f6abec6c96e770aa2e02&profile_id=113" target="_blank">
```

Comment:
@akrawchyk  We found this button style from a codrops article and it was named shikoba, so that's how I currently have it named. We can rename it as we see fit though if you think it'd be better not to have Shikoba as the class